make/test.mk

```diff
@@ -331,7 +331,7 @@ create-member2:
 ifeq ($(SECOND_MEMBER_MODE),true)
 	@echo "Preparing namespace for second member operator: ${MEMBER_NS_2}..."
 	$(MAKE) create-project PROJECT_NAME=${MEMBER_NS_2}
-	-oc label ns --overwrite=true ${MEMBER_NS_2} app=member-operator
+	-oc label ns --overwrite=true ${MEMBER_NS_2} app=member-operator toolchain.dev.openshift.com/webhook=disabled
```

Comment:
why do we need this label there? Disabling the webhook should happen through the ToolchainConfig/MemberConfig, right?

Reply:
correct, removed in https://github.com/codeready-toolchain/toolchain-e2e/pull/950/commits/9e8562ad953ad0a7571771ca473797e7ee713ce3

Thanks!